addons/attributes/functions/fnc_gui_checkbox.sqf

```diff
@@ -0,0 +1,31 @@
+#include "script_component.hpp"
+/*
+ * Author: mharis001
+ * Initializes the checkbox attribute control type.
+ *
+ * Arguments:
+ * 0: Controls Group <CONTROL>
```

Comment:
Pretty certain previously it was just `Control Group` + it sounds better

All files then should be updated in accordance with this suggestion and variables too.
```suggestion
 * 0: Control Group <CONTROL>
```

Reply:
No, it is supposed to be "controls group".